pkgs/development/ocaml-modules/phylogenetics/default.nix

```diff
@@ -1,30 +1,38 @@
-{ lib, buildDunePackage, fetchFromGitHub, ppx_deriving
-, alcotest, angstrom-unix, biocaml, gnuplot, gsl, lacaml, menhir, owl, printbox }:
+{ lib, buildDunePackage, fetchFromGitHub, ppx_deriving, bppsuite
+, alcotest, angstrom-unix, biocaml, core, gsl, lacaml, menhir, menhirLib, printbox }:
 
 buildDunePackage rec {
   pname = "phylogenetics";
-  version = "unstable-2020-11-23";
+  version = "0.0.0";
 
   useDune2 = true;
 
   src = fetchFromGitHub {
     owner  = "biocaml";
     repo   = pname;
-    rev    = "e6e10efa0a3a8fd61bf4ab69e91a09549cc1ded6";
-    sha256 = "0pmypzp0rvlpzm8zpbcfkphwnhrpyfwfv44kshvx2f8nslmksh8c";
+    rev    = "v${version}";
+    sha256 = "sha256:0h7ik1zcl10259zn3bh4gj7ax0xdxc4jm6ph9y4i3q9z09g1m2g1";
   };
```

Comment:
Please use the released source code: https://github.com/biocaml/phylogenetics/releases/download/v0.0.0/phylogenetics-0.0.0.tbz
In particular, it includes correct version information.